frontend/ui/src/pages/Inbox/ConversationListItem/index.tsx

```diff
@@ -44,7 +44,7 @@ const ConversationListItem = (props: ConversationListItemProps) => {
 
   const participant = conversation.metadata.contact;
   const unread = conversation.metadata.unreadCount > 0;
-  const currentConversationState = conversation.metadata.state;
+  const currentConversationState = conversation.metadata.state || 'OPEN';
```

Comment:
we need to add a check here because when a new conversation is created `conversation.metadata.state ` is undefined